stories/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.conf.urls import url, include
+from rest_framework.urlpatterns import format_suffix_patterns
+from stories import views
+from django.conf.urls.static import static
+from django.conf import settings
+
+urlpatterns = [
+    url(r'^auth/', include('rest_framework.urls', namespace='rest_framework')),
+    url(r'^stories/$', views.ResponsesList.as_view(), name="create"),
+    url(r'^stories/user/(?P<pk>[0-9]+)/$', views.UserStoriesView.as_view(), name="details"),
```

Comment:
Be consistent, in the view you specified the lookup field as `fb_id` but in this url you're using `id`